sinks/signalfx/signalfx_test.go

```diff
@@ -713,13 +709,14 @@ func TestSignalFxClientByTagUpdater(t *testing.T) {
 	derived := newDerivedProcessor()
 	perBatch := 1
 
-	sink, err := NewSignalFxSink("host", "glooblestoots", map[string]string{"yay": "pie"}, logrus.New(), fallback, "test_by", map[string]DPClient{}, nil, nil, derived, perBatch, "", true, 5*time.Millisecond, "", server.URL, server.Client())
+	sink, err := NewSignalFxSink("host", "glooblestoots", map[string]string{"yay": "pie"}, logrus.New(), fallback, "test_by", map[string]DPClient{}, nil, nil, derived, perBatch, "", true, dynamicKeyRefreshPeriod, "", server.URL, server.Client())
 	require.NoError(t, err)
 
 	err = sink.Start(nil)
 	require.NoError(t, err)
 
-	time.Sleep(50 * time.Millisecond)
+	// TODO better synchronization here than time.Sleep
+	time.Sleep(10 * dynamicKeyRefreshPeriod)
```

Comment:
😂 😭

Reply:
To be clear, this isn't actually a semantic change - it just switches to making the calculation explicit rather than hardcoded.

But yes.